driver-moab/main/prep_ocn_mod.F90

```diff
@@ -837,85 +837,85 @@ subroutine prep_ocn_init(infodata, atm_c2_ocn, atm_c2_ice, ice_c2_ocn, rof_c2_oc
-             if (iamroot_CPLID) then
-                write(logunit,*) ' '
-                write(logunit,F00) 'Initializing MOAB mapper_Fr2o'
-             end if
-             ! If loading map from disk, then load the scalar map as well
+            ! now take care of the mapper for MOAB mapper_Fr2o
+            if (iamroot_CPLID) then
+               write(logunit,*) ' '
+               write(logunit,F00) 'Initializing MOAB mapper_Fr2o'
+            end if
+            ! If loading map from disk, then load the scalar map as well
             if (.not. compute_maps_online_r2o) then
                type_grid = 3 ! this is type of grid
                arearead = 0 ! no need for areas
                call moab_map_init_rcfile( mbrxid, mboxid, mbintxro, type_grid, &
                      'seq_maps.rc', 'rof2ocn_fmapname:', 'rof2ocn_fmaptype:', samegrid_ro, &
                      arearead, wgtIdFr2o, 'mapper_Fr2o MOAB initialization', esmf_map_flag, wgtIdFr2ol )
 
-               context_id = rmapid ! ocn(1)%cplcompid
-               ! this creates a parallel communication graph between mbrxid and mbintxro,
-               ! with ids rof(1)%cplcompid, rmapid (rmapid is 100*src+tgt)
-               ! this will be used in send/receive mappers
-               ierr = iMOAB_MigrateMapMesh ( mbrxid, mbintxro, mpicom_CPLID, mpigrp_CPLID, &
-                                             mpigrp_CPLID, type_grid, rof(1)%cplcompid, context_id )
-               if (ierr .ne. 0) then
-                  write(logunit,*) subname,' error in migrating rof mesh for map rof c2 ocn '
-                  call shr_sys_abort(subname//' ERROR in migrating rof mesh for map rof c2 ocn ')
-               endif
-
             end if
 
-             mapper_Fr2o%src_mbid = mbrxid
-             mapper_Fr2o%tgt_mbid = mboxid ! special
-             mapper_Fr2o%intx_mbid = mbintxro
-             mapper_Fr2o%src_context = rof(1)%cplcompid
-             mapper_Fr2o%intx_context = rmapid ! read map is the same context as intersection now
-             mapper_Fr2o%weight_identifier = wgtIdFr2o
-             mapper_Fr2o%mbname = 'mapper_Fr2o'
+            mapper_Fr2o%src_mbid = mbrxid
+            mapper_Fr2o%tgt_mbid = mboxid ! special
+            mapper_Fr2o%intx_mbid = mbintxro
+            mapper_Fr2o%src_context = rof(1)%cplcompid
+            mapper_Fr2o%intx_context = rmapid ! read map is the same context as intersection now
+            mapper_Fr2o%weight_identifier = wgtIdFr2o
+            mapper_Fr2o%mbname = 'mapper_Fr2o'
 #endif
-          endif
+         endif
 
+         context_id = rmapid ! ocn(1)%cplcompid
+         ! this creates a parallel communication graph between mbrxid and mbintxro,
+         ! with ids rof(1)%cplcompid, rmapid (rmapid is 100*src+tgt)
+         ! this will be used in send/receive mappers
+         type_grid = 3 ! this is type of grid
+         ierr = iMOAB_MigrateMapMesh ( mbrxid, mbintxro, mpicom_CPLID, mpigrp_CPLID, &
```

Comment:
This should technically be within `if (.not. compute_maps_online_r2o) then` block. Even though r2o is always read from a file, the style should be consistent with the rest of the components.